src/layouts/MemberRegistration/MemberRegistration.jsx

```diff
@@ -107,6 +111,7 @@ export const MemberRegistration = () => {
 				members={members}
 				onClickBackBtn={() => {
 					setShowExistingRegistrationTable(false);
+					navigate(-1);
```

Comment:
I am not sure if this will properly redirect in all the scenarios
```suggestion
					isAdmin: navigate(PageRoutes.MEMBER_SEARCH): navigate(PageRoutes.HOMESCREEN);
```